CSS/styles-about.css

```diff
@@ -0,0 +1,489 @@
+}
+
+.Le-Slogan {
+  font-family: 'COCOGOOSE', sans-serif;
+  line-height: 55px;
+  color: transparent;
+  font-size: 50px;
+  font-weight: 600;
+  width: 100%;
+  background-image: url(../images/Le-slogan-gradient.jpg);
+  background-size: cover;
+  background-position-y: bottom;
+  -webkit-background-clip: text;
+  margin-bottom: 7px;
+}
+
+.below-slogan {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-bottom: 45px;
+}
+
+.Lemorse-quote {
+  height: auto;
+  font-size: 13px;
+  line-height: 150%;
+  padding: 1em 0.5em;
+  text-align: justify;
+  margin: 17px 0.75em 0;
+  width: calc(100% - 2.5em);
+  font-family: 'Lato', sans-serif;
+  border: 4px solid rgb(247, 247, 248);
+  background-color: rgb(255, 255, 255);
+}
+
+.small-info {
+  width: calc(100% - 2.5em);
+  margin-bottom: 53px;
+}
+
+.lemorse-mssg {
+  padding: 40px 40px 10px 40px;
+  font: 400 15px 'Lato', sans-serif;
+  color: rgb(51, 51, 51);
+  text-align: center;
+  line-height: 22px;
+  width: 90%;
+}
+
+.cntc-email {
+  font: 700 14px 'Lato', sans-serif;
+  color: rgb(51, 51, 51);
+  text-decoration: underline;
+}
+
+.underline {
+  width: 10%;
+  height: 1px;
+  background-color: #ec5242;
+  margin: 8px auto;
+}
+
+.moreline {
+  width: 50%;
+}
+
+.memorable-logo {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+  border: 1px solid #e4e4e4;
+  padding: 51px 25px 0 25px;
+}
+
+.memorable-logo-tt {
+  text-align: center;
+  font-size: 24px;
+  font-family: 'Lato', sans-serif;
+}
+
+.competition-logo {
+  font-family: 'Lato', sans-serif;
+  text-align: center;
+  margin: 0;
+  font-size: 16px;
+  line-height: 24px;
+  color: #333;
+}
+
+.big-logo {
+  width: 75%;
+  height: 20%;
+  margin: 55px;
+  border: 1px solid #e4e4e4;
+}
+
+.previous-Researches {
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  display: flex;
+  margin: 20px;
+  gap: 40px;
+}
+
+.previous-Researches h2 {
+  font-family: 'Lato', sans-serif;
+  text-align: center;
+}
+
+.previous-Researches p {
+  font-family: 'Lato', sans-serif;
+  text-align: center;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.research-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 30px;
+  gap: 15px;
+  width: 100%;
+}
+
+.research-date {
+  font-family: 'COCOGOOSE', sans-serif;
+  font-size: 50px;
+  font-weight: 700;
+}
+
+.research-1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 5px;
+  background-image: url(../images/research-img-R1.jpg);
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  box-shadow: inset 0 0 0 100px rgb(239 81 60 / 67%);
+  width: 80%;
+  height: 200px;
+  color: #fff;
+}
+
+.research-2 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 5px;
+  background-image: url(../images/research-img-R2.png);
```

Comment:
- [OPTIONAL] I think it's better to use an image with the same proportion of width/height for both types of research. This little difference is presenting some gaps on the top and bottom.

Reply:
Everything is okay the animation of box shadow is making the supposed "gap" so it is actually fine, but thanks for letting me know that ElsonOtake😉